projects/valtimo/dossier-management/src/lib/components/dossier-management-statuses/dossier-management-status-modal/dossier-management-status-modal.component.scss

```diff
@@ -27,6 +27,10 @@
   flex-direction: row;
   gap: 16px;
   align-items: flex-end;
+
+  input:read-only {
+    color: var(--cds-text-disabled);
```

Comment:
There's read only in this stylesheet that you should use: https://github.com/valtimo-platform/valtimo-frontend-libraries/blob/development/12.0.0/projects/valtimo/components/assets/css/compatibility.scss

Reply:
I discussed this with Peet. The field should be disabled. Unfortunately, that isn't possible for technical reasons: validation isn't done for fields that are disabled. So I made the field read-only and made it look like its disabled